pwnlib/elf/elf.py

```diff
@@ -1753,7 +1753,8 @@ def canary(self):
     @property
     def packed(self):
         """:class:`bool`: Whether the current binary is packed with UPX."""
-        return b'UPX!' in self.get_data()
+        if b'UPX!' in self.get_data()[:0xFF]:
+            return True
```

Comment:
The way this is written now would have the function return `None` of it's not packed. The previous direct return syntax was better in this regard.

Reply:
agreed , should do now ig, lemme know if you got any other suggestions about the code